zerver/openapi/zulip.yaml

```diff
@@ -13271,14 +13271,14 @@ paths:
                             {
                               "name": "google",
                               "display_name": "Google",
-                              "display_icon": "/static/images/landing-page/logos/googl_e-icon.png",
+                              "display_icon": "/static/images/auth-methods/googl_e-icon.png",
                               "login_url": "/accounts/login/social/google",
                               "signup_url": "/accounts/register/social/google",
                             },
                             {
                               "name": "github",
                               "display_name": "GitHub",
-                              "display_icon": "/static/images/landing-page/logos/github-icon.png",
+                              "display_icon": "/static/images/auth-methods/github-icon.png",
```

Comment:
I'm not thrilled with this name just because it doesn't have a lot in common with `zproject/backends.py`. I've been wanting to rename that to `authentication_backends.py`, but it's awkward to do so because the filename is hardcoded in self-hosted servers' configuration files. Though maybe `authentication_methods.py` would be a better name there.

This part of the codebase seems to be big into using `-`s as separators rather than `_`s, but we're overall about 50/50 between the two for both images and HTML templates, and have been trying to move towards using `_`s for most filenames for static assets. So maybe `authentication_backends` or `authentication_methods` would be least likely to need to change again?

Reply:
Okay. I went with `authentication_backends` for the directory name.